bundles/persistence/org.openhab.persistence.jdbc/pom.xml

```diff
@@ -0,0 +1,290 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns="http://maven.apache.org/POM/4.0.0"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+	<parent>
+		<groupId>org.openhab.bundles</groupId>
+		<artifactId>persistence</artifactId>
+		<version>1.8.0-SNAPSHOT</version>
+	</parent>
+
+	<name>openHAB JDBC SQL Persistence</name>
+
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>org.openhab.persistence</groupId>
+	<artifactId>org.openhab.persistence.jdbc</artifactId>
+
+	<packaging>eclipse-plugin</packaging>
+
+	<properties>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+		<hikari.version>2.4.1</hikari.version>
+		<!-- <hikari.version>2.3.9</hikari.version> <hikari.version>2.4.1</hikari.version> -->
+		<dbutils.version>1.6</dbutils.version>
+		<yank.version>3.0.3-SNAPSHOT</yank.version>
```

Comment:
We will get a [new yank release 3.1.0](https://github.com/timmolter/Yank/issues/27#issuecomment-159532252) on maven central by timmolter.